### PR TITLE
Polish PayID Demo

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Add PayID demos + docs 

Note: This PR will be held for go-live. We'll also replicate to Java / Swift when we're happy with the context here. 

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

**Comment by sappenin** [comment]
Do we want the payid to show `dev` in [node-payid-demo.png]:(https://github.com/xpring-eng/Xpring-SDK-Demo/pull/38/files?file-filters%5B%5D=.js&file-filters%5B%5D=.png#diff-377f318eb59491f7a85c6f1365c6a539)

File: node/src/index-payid.js [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
 
-// A client for PayID
-const payIDClient = new XRPPayIDClient(network);
+// The BTC network to resolve on.
+const btcNetwork = 'btc-testnet'
+
+// A client to resolve PayIDs on the XRP Ledger.
+const xrpPayIdClient = new XRPPayIDClient(xrpNetwork);
+
+// A client to resolve PayIDs on the Bitcoin network.
```
Comment (hbergren) [inline node/src/index-payid.js:15]:
```suggestion
// A client to resolve PayIDs on the Bitcoin testnet.
```

File: node/src/index-payid.js [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+const btcPayIdClient = new PayIDClient(btcNetwork)
 
 async function main() {
-    console.log("Resolving Pay ID: " + payID);
-    console.log("On network: " + networkToString(network))
+    console.log("Resolving Pay ID: " + payId);
```
Comment (hbergren) [inline node/src/index-payid.js:19]:
```suggestion
    console.log("Resolving PayID: " + payId);
```

File: node/src/index-payid.js [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
+    console.log("On network: " + networkToString(xrpNetwork))
 
-    const xrpAddress = await payIDClient.xrpAddressForPayID(payID)
+    const xrpAddress = await xrpPayIdClient.xrpAddressForPayID(payId)
     console.log("Resolved to " + xrpAddress)
+    console.log("")
+
+    console.log("Resolving Pay ID: " + payId);
```
Comment (hbergren) [inline node/src/index-payid.js:26]:
```suggestion
    console.log("Resolving PayID: " + payId);
```

**Pull request merged**